product_details/FirefoxEnterprise.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Firefox Enterprise",
+  "description": "Firefox Enterprise browser",
+  "home_page_sort": 5,
+  "featured_versions": ["auto"],
+  "in_buildhub": false,
+  "bug_links": [
+    [
+      "Firefox Enterprise",
+      "https://bugzilla.mozilla.org/enter_bug.cgi?bug_type=%(bug_type)s&keywords=crash&product=Firefox+Enterprise&op_sys=%(op_sys)s&rep_platform=%(rep_platform)s&cf_crash_signature=%(signature)s&short_desc=%(title)s&comment=%(description)s&format=__default__"
+    ],
```

Comment:
The JSON format doesn't allow for trailing commas in an arrary. Some JSON parsers are more liberal, but the Python JSON parser will choke on this.

Reply:
Thanks, should be better.